doc/MANUAL.md

```diff
@@ -1329,6 +1329,9 @@ The scanner examines the `Package.resolved` file which is created by the package
 The scanner examines the `PKG-INFO` and `METADATA` files for an installed Python package to extract the component name and version which
 are used to search the database for vulnerabilities.
 Support for scanning the `requirements.txt` file generated by pip is also present.
+#### Support for Version Range (`~=`) in `requirements.txt`
+The CVE binary tool now supports version ranges specified with `~=` in your `requirements.txt` file. This allows you to define a compatible version range for your Python project.
+The CVE binary tool utilizes `pip install --dry-run` to seamlessly handle and interpret version ranges specified with `~=`. This ensures effective analysis of potential updates without the need for manual parsing of `~=`. 
```

Comment:
```suggestion
The CVE binary tool utilizes `pip install --dry-run` to seamlessly handle and interpret version ranges specified with `~=`. This ensures effective analysis of potential updates without the need for manual parsing of `~=`. 
Note that `--dry-run` will give "what you get if you installed this requirements list today" so that's what the scan results will show.  We do not attempt to retrieve CVEs for all possible versions in that range.  If you want to check a specific version within that range (such as the very first allowed one) you would need to use a separate requirements.txt file with that version frozen.
```